lib/models/builder.js

```diff
@@ -4,10 +4,11 @@ var fs          = require('fs');
 var path        = require('path');
 var Promise     = require('../ext/promise');
 var rimraf      = Promise.denodeify(require('rimraf'));
-var ncp         = Promise.denodeify(require('ncp'));
 var Task        = require('./task');
 var SilentError = require('../errors/silent');
 var chalk       = require('chalk');
+var cpd         = require('ember-cli-copy-dereference');
+var mkdirp      = require('mkdirp');
```

Comment:
`fs-extra` is a dependency which has `fs.mkdirs`. It does `mkdir -p` by default.